fixtures/user.js

```diff
@@ -0,0 +1,58 @@
+const sqlFixtures = require('sql-fixtures');
+const cleandb = require('./cleandb');
+
+const dbConfig = {
+  client: 'pg',
+  connection: {
+    host: 'localhost',
+    user: 'ferret',
+    password: '',
+    database: 'test',
+    port: 5432,
+  },
+};
+
+const dataSpec = {
+  users: [
+    {
```

Comment:
I would create a `utils.js` file in this folder (the exception from my rule with the content of this folder) containing helping functions. The only function you would have for now is probably one that accepts as parameters values for user's fields and returns an object like this one. Most of the fields can be defined in that function with default values thus making your work easier.